error_test.go

```diff
@@ -0,0 +1,61 @@
+package microerror
+
+import (
+	"fmt"
+	"path/filepath"
+	"testing"
+
+	"github.com/juju/errgo"
+)
+
+func TestMask_Nil(t *testing.T) {
+	err := Mask(nil)
+	if err != nil {
+		t.Fatalf("expected nil, got %v", err)
+	}
+}
+
+func TestMaskf_Nil(t *testing.T) {
+	err := Maskf(nil, "test annotation")
+	if err != nil {
+		t.Fatalf("expected nil, got %v", err)
+	}
+}
+
+func TestMask_Location(t *testing.T) {
+	err := fmt.Errorf("test")
+
+	err = Mask(err)
+	err = Mask(err)
+	err = Mask(err)
+
+	errgoErr, ok := err.(*errgo.Err)
+	if !ok {
+		t.Fatalf("expected type *errgo.Err, got %T", err)
+	}
+
+	file := filepath.Base(errgoErr.Location().File)
+	wfile := "error_test.go"
+	if file != wfile {
+		t.Fatalf("expected  %s, got %s", wfile, file)
+	}
+}
+
+func TestMaskf_Location(t *testing.T) {
+	err := fmt.Errorf("test")
+
+	err = Maskf(err, "1")
+	err = Maskf(err, "2")
+	err = Maskf(err, "3")
+
+	errgoErr, ok := err.(*errgo.Err)
+	if !ok {
+		t.Fatalf("expected type *errgo.Err, got %T", err)
+	}
+
+	file := filepath.Base(errgoErr.Location().File)
+	wfile := "error_test.go"
+	if file != wfile {
+		t.Fatalf("expected  %s, got %s", wfile, file)
+	}
```

Comment:
@xh3b4sd fixed with https://github.com/giantswarm/microerror/pull/3/commits/41248ae08def8c90f32a82c8ae1fde9f5a66cbf5

Strange because it was copy/paste from microkit.

Current output is:
```
[{/Users/pawel/go/src/github.com/giantswarm/microerror/pawel/main.go:14: 3} {/Users/pawel/go/src/github.com/giantswarm/microerror/pawel/main.go:13: 2} {/Users/pawel/go/src/github.com/giantswarm/microerror/pawel/main.go:12: 1} {test}]
```
 Can be easily changed to the another one from the comment above.